### ISSUE TITLE
Add Keras Normalization layer

### ISSUE THREAD

**Issue by pobonomo** [Gurobi/gurobi-machinelearning#10]
It should be indentical to sklearn's StandardScaling
